src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -93,13 +93,18 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
           <div id="tooltip">
             <div class="title">${computeStateName(stateObj)}</div>
             <div class="current-temperature">
-              <span class="current-temperature-text">${
-                stateObj.attributes.current_temperature
-              }
-                <span class="uom">${
-                  this.hass.config.unit_system.temperature
-                }</span>
-              </span>
+            ${
+              stateObj.attributes.current_temperature
+                ? html`
+                <span class="current-temperature-text">
```

Comment:
Take out the surrounding Span of `current-temperature-text` and only have 
```
${stateObj.attributes.current_temperature}
                }</span>	                  <span class="uom">
              </span>	                    ${this.hass.config.unit_system.temperature}
                  </span>
```

This be the rendered or not rendered part. and then in the else just ``` html`` ```